src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -37,11 +37,45 @@ const elementEventBindingDataList: ElementEventBindingData[] = [
         name: "mouseleave",
     },
     {
-        eventID: EventID.SCROLL_INTO_VIEW,
-        name: "scrollintoview",
-    },
-    {
-        eventID: EventID.SCROLL_OUT_OF_VIEW,
-        name: "scrolloutofview",
+        addDOMEvent: (elementSelectionBehaviour: ElementSelectionBehaviour) => {
```

Comment:
I agree it is very ugly, but I don't think moving these pretty specific functions to global scope, this would result in equally ugly code. Any suggestions are welcome, I have put all my refactoring skills in there already.